components/chart/index.tsx

```diff
@@ -56,11 +57,36 @@ function Chart({
     ZOOM_IN: () => changeZoomLevel(-10),
     ZOOM_OUT: () => changeZoomLevel(10),
   };
+  useEffect(() => {
+    const queryParams = parse(window.location.search);
+    const nValue = queryParams['n'];
+    if (nValue) {
+      const nodeId = parseInt(nValue as string);
+      setSelectedNodeID(nodeId);
+    } else {
+      setSelectedNodeID(null);
+    }
+  }, []);
+  useEffect(() => {
+    if (selectedNodeID !== null) {
+      const node = findModuleById(selectedNodeID);
+      selectModule(node);
+    }
+  }, [selectedNodeID]);
+
+  const updateURLWithNodeID = (nodeID: number | null) => {
+    const queryParams = parse(window.location.search);
+    queryParams['n'] = nodeID !== null ? String(nodeID) : '';
+    const newQueryString = stringify(queryParams);
+    const newURL = `${window.location.pathname}?${newQueryString}`;
+    window.history.replaceState({}, '', newURL);
+    localStorage.setItem('selectedNodeID', nodeID !== null ? String(nodeID) : '');
+  };
```

Comment:
I don't feel we need to store the selectedNodeID inside local storage, just whenever the user clicks on share link, we can build up the link and copy it
You can use a library like the next one,
https://www.npmjs.com/package/copy-to-clipboard
I felt you wanted to do both #13 and #14 in the same MR which is okay if you want

Reply:
hmmm I thought about it again, I think no need to use that library, you can use the built-in Clipboard API in the browser

Reply:
done

Reply:
In this case why we don't move this function to description component, I think no need to keep it here, and we need to update it's name to be more clear